backend/Platform/settings/production.py

```diff
@@ -43,3 +43,10 @@
 EMAIL_USE_TLS = True
 
 IS_DEV_LOGIN = os.environ.get("DEV_LOGIN", "False") in ["True", "TRUE", "true"]
+
+# AWS S3
+DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
```

Comment:
Haven't used s3boto3Storage before - does this basically automatically upload the images to s3 once they're on our server? I don't see any other logic for s3 uploading so wanted to make sure. Secondarily if this is the case, when someone uploads a new profile picture, how does the deletion of the old one on s3 get handled since we don't want to clutter s3 up unnecessarily?

Reply:
My understanding is that the behavior with s3 uploading is the same as if the files were stored locally, so when a new profile picture is uploaded the old photo is just deleted from s3